autoload/neomake/makers/ft/python.vim

```diff
@@ -83,6 +83,11 @@ function! neomake#makers#ft#python#Flake8EntryProcess(entry) abort
     else
         let type = ''
     endif
+    let l:token = matchstr(a:entry.text, "'.*'")
+    if strlen(l:token)
+        let a:entry.length = strlen(l:token) - 2 " subtract the quotes
+    endif
```

Comment:
Oh, it was not clear to me (and still is not): then length will only be set if the text is in quotes?
How does entry.length get set in general?  (is it only overwritten here?)